### PR TITLE
338 update slc community app notebook

### PR DESCRIPTION
**Pull request by Vismayak**

### CODE REVIEW

File: notebooks/slc_community_app.ipynb [hunk 1]
```diff
@@ -1488,7 +1488,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.8"
+   "version": "3.12.2"
```
Comment (ywkim312) [inline notebooks/slc_community_app.ipynb:1491]: Just curious. What is the reason for bumping up the version of this?

Comment (navarroc) [reply notebooks/slc_community_app.ipynb:1491]: My guess is he opened the notebook and it recorded his version of python from his environment. It's just metadata.

File: notebooks/retrofit_data/SLC_plus_retrofit.json
Comment (navarroc) [inline notebooks/retrofit_data/SLC_plus_retrofit.json:1]: Is this file and the CSV needed in this PR? I don't see them being used.

Comment (Vismayak) [reply notebooks/retrofit_data/SLC_plus_retrofit.json:1]: Yup, my bad! I forgot to remove them after I uploaded them to the server, thank you!

**Review by Rashmil-1999 — APPROVED** [review]
It runs fine and everything looks good

**Comment by navarroc** [comment]
@Vismayak can you clear the outputs from the retrofit notebook?

**Review by navarroc — APPROVED** [review]

**Pull request merged**